src/components/RecipeForm/RecipeForm.styled.tsx

```diff
@@ -0,0 +1,343 @@
+import { Divider, styled } from "@mui/material";
+import submitButtonImage from "../../images/submitButton.png";
+
+export const StyledFormLineContainer = styled("div")`
+  max-width: 95%;
+  display: flex;
+  margin-top: ${({ theme }) => theme.spacing(2)};
+  align-items: flex-end;
+  flex-wrap: nowrap;
+`;
+
+export const StyledFormLineWrapContainer = styled("div")`
+  max-width: 95%;
+  display: flex;
+  margin-top: ${({ theme }) => theme.spacing(2)};
+  align-items: flex-end;
+  flex-wrap: wrap;
+`;
+
+export const StyledFormContainer = styled("div")`
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  background-color: ${({ theme }) => theme.palette.secondary.light};
+  padding: ${({ theme }) => theme.spacing(8)} ${({ theme }) => theme.spacing(8)};
+  margin: ${({ theme }) => theme.spacing(8)} ${({ theme }) => theme.spacing(14)};
+
+  ${({ theme }) => theme.breakpoints.down("lg")} {
+    padding: ${({ theme }) => theme.spacing(12)}
+      ${({ theme }) => theme.spacing(6)};
+    margin: ${({ theme }) => theme.spacing(12)}
+      ${({ theme }) => theme.spacing(10)};
+  }
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    padding: ${({ theme }) => theme.spacing(10)}
+      ${({ theme }) => theme.spacing(4)};
+    margin: ${({ theme }) => theme.spacing(10)}
+      ${({ theme }) => theme.spacing(8)};
+  }
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    padding: ${({ theme }) => theme.spacing(8)}
+      ${({ theme }) => theme.spacing(2)};
+    margin: ${({ theme }) => theme.spacing(8)}
+      ${({ theme }) => theme.spacing(4)};
+  }
+
+  button {
+    font-family: "Playfair Display", serif;
+    font-size: 24px;
+    min-height: 72px;
+    ${({ theme }) => theme.breakpoints.down("lg")} {
+      font-size: 20px;
+      min-height: 54px;
+    }
+    ${({ theme }) => theme.breakpoints.down("md")} {
+      font-size: 18px;
+      min-height: 40px;
+    }
+    ${({ theme }) => theme.breakpoints.down("sm")} {
+      font-size: 14px;
+      min-height: 36px;
+    }
+  }
+
+  label {
+    font-family: "Playfair Display", serif;
+    font-size: 24px;
+
+    ${({ theme }) => theme.breakpoints.down("lg")} {
+      font-size: 20px;
+    }
+    ${({ theme }) => theme.breakpoints.down("md")} {
+      font-size: 18px;
+    }
+    ${({ theme }) => theme.breakpoints.down("sm")} {
+      font-size: 14px;
+    }
+  }
+
+  input,
```

Comment:
why are you using global selectors instead of styled components for each element?